docs/pywbemcligeneraloptions.rst

```diff
@@ -1,25 +1,34 @@
 .. _`Pywbemcli command line general options`:
 
-Pywbemcli command line general options
-======================================
+Using the pywbemcli command line general options
+================================================
 
 
-.. _`General options overview`:
+.. _`Oveview of the general options`:
 
-General options overview
-------------------------
+Overview of the general options
+-------------------------------
 
-The general options are entered before the command group or command. They
-define:
+General options must be entered on the command line  before the command group
+or command. They define:
 
 * Characteristics of the WBEM server against which the commands are to be
-  executed (i.e url, default-namespace, security parameters, etc.)
-* Execution options that apply to multiple commands (i.e. output
-  formats, statistics keeping, logging and using pull operations).
-* General options to show version, display additional information.
+  executed (i.e url, default-namespace, security parameters, etc.).
+  See :ref:`Defining the WBEM Server`
+* Operation behavior of pywbemtools for requests; i.e. using the pull vs.
+  the :term:`traditional operations` and defined in
+  :ref:`Pywbemcli and the DMTF pull operations`, and the client side timeout.
+  See :ref:`Controlling operation behavior and monitoring operations`
+* Execution options to monitor the requests and responses(statistics keeping, and
+  logging of operations) and to display the results like output formats.
+  See :ref:`Controlling operation behavior and monitoring operations`  and
+  :ref:`Controlling result output formats`
+* General options to show the pywbemtools version, and display help for any of
+  the command groups or commands.
 
 For example the following enumerates the qualifier declarations and outputs the
-result as a ``simple`` table:
+result as a ``simple`` table because the ``--output-format defines the output
+as ``simple``, the simple table format:
```

Comment:
This sentence could be streamlined somewhat.
Also, backquotes missing after --output-format.

Reply:
Done.